portability-transfer/src/main/java/org/datatransferproject/transfer/WorkerMain.java

```diff
@@ -77,7 +77,7 @@ public void initialize() {
     // TODO this should be moved into a service extension
     extensionContext.registerService(HttpTransport.class, new NetHttpTransport());
     extensionContext.registerService(OkHttpClient.class, new OkHttpClient.Builder().build());
-    extensionContext.registerService(JsonFactory.class, new JacksonFactory());
+    extensionContext.registerService(JsonFactory.class, GsonFactory.getDefaultInstance());
```

Comment:
I have a slight concern with updating this in the extension context because it gets pulled in arbitrarily in other classes.. ex: it gets piped through to the GooglePhotosInterface through this line: https://github.com/google/data-transfer-project/blob/889f1fa92fc54046b42a9448b53d9de0df84912c/extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/GoogleTransferExtension.java#L76,L109 

I'm not sure how this would behave if another service is using Jackson somewhere else in their code but pulling in this Gson factory instead. 

I would guess there shouldnt be too many instances of this but i have not done the analysis

Reply:
I updated GooglePhotosInterface to use the provided `JsonFactory` so it will at least be consistent with whatever is done here.